src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -202,22 +202,35 @@ private function registerBusToCollector(ContainerBuilder $container, string $bus
 
     private function registerBusMiddleware(ContainerBuilder $container, string $busId, array $middleware)
```

Comment:
Could you rename to `$middlewareCollection` to make it more explicit?

Reply:
Sure, good idea 👍 (uncountable nouns are hell in programmation)